all.json

```diff
@@ -1501,9 +1501,9 @@
 		"comb.fund",
 		"combinewallet.com",
 		"community-airdrop-program.com",
-		"community.trustwallet.com-recovery.45-88-108-231.plesk.page",
-		"community.trustwallet.com.45-88-108-231.plesk.page",
-		"compassionate-lalande.96-9-210-124.plesk.page",
+		"45-88-108-231.plesk.page",
+		"45-88-108-231.plesk.page",
```

Comment:
This is actually one of the reasons we are doing this - after listing the first and before getting to the second, we may have missed a beat.